lib/perl/Genome/Disk/Detail/Allocation/Mover.pm

```diff
@@ -30,6 +30,22 @@ class Genome::Disk::Detail::Allocation::Mover {
     ],
 };
 
+sub __errors__ {
+    my $self = shift;
+
+    my @errors = $self->SUPER::__errors__;
+    return @errors if @errors;
+
+    if ( $self->disk_group_name and $self->target_mount_path ) {
+        push @errors, UR::Object::Tag->create(
+            type => 'error',
+            properties => [qw/ group_name target_mount_path /],
+            desc => 'Can not specify both target group and volume!', 
```

Comment:
@tmooney I wanted to do that, but the whole interface is strings, and not objects. In order to check them, the objects would need to be retrieved. I think that would be better, but it seemed that it should be done higher up, and I wanted to keep the PR small. It would be much trouble to update the commands and allocation to pass objects around. What do you think?